critiquebrainz/db/review.py

```diff
@@ -40,6 +40,22 @@ def to_dict(review, confidential=False):
     return review
 
 
+def get_top_languages():
+    """
+        List of the top ten languages (only the language codes) in CritiqueBrainz.
+    """
```

Comment:
Make sure that the docstring matches the guidelines: https://github.com/metabrainz/guidelines/blob/master/Python.md#docstrings